src/coreclr/jit/gentree.cpp

```diff
@@ -20760,33 +20760,55 @@ GenTree* Compiler::gtNewSimdSumNode(
         case TYP_UBYTE:
         case TYP_SHORT:
         case TYP_USHORT:
+        {
+            tmp = gtNewSimdAsHWIntrinsicNode(simdType, op1, NI_AdvSimd_Arm64_AddAcross, simdBaseJitType, simdSize);
+            return gtNewSimdAsHWIntrinsicNode(type, tmp, NI_Vector64_ToScalar, simdBaseJitType, 8);
+        }
+
         case TYP_INT:
         case TYP_UINT:
         {
-            tmp = gtNewSimdAsHWIntrinsicNode(simdType, op1, NI_AdvSimd_Arm64_AddAcross, simdBaseJitType, simdSize);
+            if (simdSize == 8)
+            {
+                op1 = gtNewSimdHWIntrinsicNode(TYP_SIMD16, op1, NI_Vector64_ToVector128, simdBaseJitType, simdSize,
```

Comment:
Not sure I understand why we can't do `AddPairwise(op1, op1Clone).ToScalar()` here?

Reply:
We should be able to. I just went with the first thing that came to mind. It's been a long day trying to ensure this could be completed for the snap so the feature previews working as intended 😅

Reply:
If possible, I'd like to get this in as a follow up PR (assuming CI passes and I don't have to push anything else).

It's a trivial change and something I can push immediately after; just want to ensure things are in and functioning generally as intended.

Reply:
I am okay with that.

Reply:
Ended up fixing this as `ExtractMostSignificantBits` needed a similar fix.